virt/virt-4-13-release-notes.adoc

```diff
@@ -164,6 +164,11 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-18090 Release note: PREVIEW OVN Secondary Network
 
 //CNV-18095 Release note: NEW [TechPreview] Add Virtual machines CPU metrics
+* You can now use xref:../virt/support/monitoring/virt-prometheus-queries.adoc#about-querying-metrics_virt-prometheus-queries[Prometheus] to monitor the following metrics:
+
+** `kubevirt_vmi_cpu_system_seconds` returns the physical system CPU time consumed by the hypervisor.
+** `kubevirt_vmi_cpu_user_seconds` returns the physical user CPU time consumed by the hypervisor.
+** `kubevirt_vmi_cpu_usage_seconds` returns the total CPU time used in seconds by calculating the sum of the vCPU and the hypervisor usage.
```

Comment:
@danielclowers The metrics names are:
kubevirt_vmi_cpu_system_usage_seconds,
kubevirt_vmi_cpu_user_usage_seconds,
kubevirt_vmi_cpu_usage_seconds